### ISSUE TITLE
IVY: properly support parsing namespaced attributes in template

### ISSUE THREAD

**Issue by benlesh** [angular/angular#24426]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior

Currently, ivy will not parse the following into the proper instructions:

```html
div xmlns:foo="http://someurl/foo" class="my-app" foo:bar="baz" title="Hello">Hello <b>World</b>!</div>
```

## Expected behavior

The instructions should look something like:

```ts
$r3$.ɵE(0, 'div', $e0_attrs$);
$r3$.ɵT(1, 'Hello ');
$r3$.ɵE(2, 'b');
$r3$.ɵT(3, 'World');
$r3$.ɵe();
$r3$.ɵT(4, '!');
$r3$.ɵe();
```

## Minimal reproduction of the problem with instructions

Create a component with the template above, examine the output of the component to check the `namespaceURI` of the `foo:bar` attribute.

## What is the motivation / use case for changing the behavior?

To support namespaced attributes in Angular.

## Environment

<pre><code>
Angular version: Ivy

Browser:
- [x] Chrome (desktop) version XX
- [x] Chrome (Android) version XX
- [x] Chrome (iOS) version XX
- [x] Firefox version XX
- [x] Safari (desktop) version XX
- [x] Safari (iOS) version XX
- [x] IE version XX
- [x] Edge version XX
 
For Tooling issues:
N/A

Others:

</code></pre>

**Comment by mhevery** [comment]
closed by #24386

**Comment by kara** [comment]
This work was never completed. Test still turned off and failing: https://github.com/angular/angular/blob/master/packages/compiler-cli/test/compliance/r3_compiler_compliance_spec.ts#L173
